extensions/wikia/FilePage/FilePageHooks.class.php

```diff
@@ -199,6 +199,10 @@ public static function onUndeleteComplete( Title $title ) {
 	 * @return true -- because it's hook
 	 */
 	public static function onArticleSave( WikiPage $page ) {
+		if(get_class($page) == "WikiFilePage" && $page->getFile()->media_type &&
```

Comment:
```suggestion
		if( get_class( $page ) == "WikiFilePage" && $page->getFile()->media_type &&
```

Reply:
Usually when checking what type of page we are dealing with we do something like this:
```php
if( $page->getTitle()->getNamespace() == NS_FILE ) {
    $page->getFile();
}
```